muxdb/cache.go

```diff
@@ -0,0 +1,201 @@
+// Copyright (c) 2021 The VeChainThor developers
```

Comment:
is this correct date ?

Reply:
The copyright year was set to the year that created that file. Does this makes sense to you?

Reply:
Based on a previous discussion and approach we implemented a dynamic range from the year that the file was created to the current year.

See https://github.com/vechain/thor/pull/693/files

Reply:
I would consider they are different stuff.

Reply:
weird it is showing like a new file for me thats why a question :)

Reply:
I see, it was moved from `muxdb/internal/trie/cache.go`